packages/amplication-plugin-api/project.json

```diff
@@ -63,6 +63,18 @@
         }
       }
     },
+    "refresh:plugins": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "curl --location --request POST 'http://localhost:3005/graphql' --header 'Content-Type: application/json' --data-raw '{\"query\":\"mutation ProcessPluginCatalog {processPluginCatalog {  name  npm  versions {    version    updatedAt    deprecated  }}}\"}'",
+        "envFile": "packages/amplication-plugin-api/.env"
+      },
+      "configurations": {
+        "production": {
+          "command": "curl --location --request POST 'https://plugin-api.amplication.com/graphql' --header 'Content-Type: application/json' --data-raw '{\"query\":\"mutation ProcessPluginCatalog {processPluginCatalog {  name  npm  versions {    version    updatedAt    deprecated  }}}\"}'"
+        }
+      }
+    },
```

Comment:
can you explain what are these scripts and why we need them?
do we need to run them while running locally?

Reply:
Those scripts will simplify the pre-population of plugin data in the pluginapi db. I'll add some explaination in the README